jobs/regular/remind_user.rb

```diff
@@ -0,0 +1,13 @@
+module Jobs
+  class RemindUser < Jobs::Scheduled
+    sidekiq_options queue: 'low'
+
+    def execute(args)
+      raise Discourse::InvalidParameters.new(:user_id) unless args[:user_id].present?
+
+      user = User.find(args[:user_id])
```

Comment:
You'll want to use `find_by` here instead and raise a `Discourse::InvalidParameters` error if the `user_id` is no longer valid.